### ISSUE TITLE
stride::get() return for fixed/static values

### ISSUE THREAD

**Issue by psiha** [jfalcou/kiwaku#40]
concretely the line:

`return std::integral_constant<size_type,1>{};`

returns always one - shouldn't it return the static value for the given I?

**Comment by psiha** [comment]
Oh i get it, i think - for strides you save static information/strides only for unit strides - could we transform this into a feature request then
- that strides save all possible fixed/static values - like shapes already do?
- and that they provide also provide an as_array() function?
- and some easy way of permuting them - creating a 'copy' with permuted values based on an index map or something similar
?
😊

**Comment by jfalcou** [comment]
Yes that's planned. I have to check, it's maybe even done in the upcoming table PR.

**Comment by jfalcou** [comment]
Fixed in #50

**Comment by jfalcou** [comment]
stride and shape now share the same internal basic implementation so get on stride is correct on any dimension and support arbitrary constant value on any dimensions
